lib/bundles/inspec-compliance/target.rb

```diff
@@ -39,11 +39,11 @@ def resolve(target, opts = {})
     end
 
     # extracts profile name from url
-    def get_profile_name(uri)
+    def self.get_profile_name(uri)
       uri.host + uri.path
     end
 
-    def build_target_url(target)
+    def self.build_target_url(target)
       owner, profile = target.split('/')
       config = Compliance::Configuration.new
       url = "#{config['server']}/owners/%owner_name%/compliance/%profile_name%/tar"
```

Comment:
This is a little weird. Why not just this (instead of line 49-52)

`"#{config['server']}/owners/#{owner}/compliance/#{profile}/tar"`?

I know this isn't part of the change set but fixing it here is as good a time as any ;)